src/Interop/Interop.csproj

```
@@ -0,0 +1,34 @@
+﻿	<Project Sdk="Microsoft.NET.Sdk">
```

indent is slightly off
```suggestion
<Project Sdk="Microsoft.NET.Sdk">
```